lib/ads.js

```diff
@@ -1330,3 +1336,60 @@ exports.NOTIFY = {
   CYCLIC: 3,
   ONCHANGE: 4
 }
+
+// ADS reserved index groups
+var ADSIGRP = {
+  SYMTAB:             0xF000,
+  SYMNAME:            0xF001,
+  SYMVAL:             0xF002,
+  SYM_HNDBYNAME:      0xF003,
```

Comment:
Let's use more clear constant names (taken from TwinCAT documentation)
```
GET_SYMHANDLE_BYNAME:    0xF003,
READ_SYMVAL_BYNAME:      0xF004,
RW_SYMVAL_BYHANDLE :     0xF005,
RELEASE_SYMHANDLE:     	 0xF006,
```

Reply:
the names are from the TcAdsDef.h 
#define ADSIGRP_SYMTAB						0xF000
#define ADSIGRP_SYMNAME						0xF001
#define ADSIGRP_SYMVAL						0xF002
...

The object was named ADSIGRP so its: ADSIGRP.SYMTAB
I can change it.. no Problem

Reply:
so it was compatible to the examples c++ for me:
https://infosys.beckhoff.de/content/1033/tcsample_vc/html/tcadsdll_api_cpp_sample12.htm?id=3900754293489007423

Reply:
No problem

Reply:
The others names ok?

Reply:
Jep, the rest is perfect